### ISSUE TITLE
ARM64 R2R: Assertion failed 'varDsc->lvOnFrame'

### ISSUE THREAD

**Issue by kyulee1** [dotnet/coreclr#5639]
JIT\jit64\hfa\main\testA\hfa_sd2A_d\hfa_sd2A_d.cmd
JIT\jit64\hfa\main\testA\hfa_sd2A_r\hfa_sd2A_r.cmd
JIT\jit64\hfa\main\testA\hfa_sf2A_d\hfa_sf2A_d.cmd
JIT\jit64\hfa\main\testB\hfa_sd2B_d\hfa_sd2B_d.cmd
JIT\jit64\hfa\main\testA\hfa_sf2A_r\hfa_sf2A_r.cmd
JIT\jit64\hfa\main\testB\hfa_sf2B_d\hfa_sf2B_d.cmd
JIT\jit64\hfa\main\testB\hfa_sf2B_r\hfa_sf2B_r.cmd
JIT\jit64\hfa\main\testB\hfa_sd2B_r\hfa_sd2B_r.cmd
JIT\jit64\hfa\main\testC\hfa_sd2C_r\hfa_sd2C_r.cmd
JIT\jit64\hfa\main\testC\hfa_sd2C_d\hfa_sd2C_d.cmd
JIT\jit64\hfa\main\testC\hfa_sf2C_r\hfa_sf2C_r.cmd
JIT\jit64\hfa\main\testC\hfa_sf2C_d\hfa_sf2C_d.cmd
```
Assert failure(PID 17956 [0x00004624], Thread: 17832 [0x45a8]): Assertion failed 'varDsc->lvOnFrame' in 'DomainBoundILStubClass:IL_STUB_PInvoke(float,double,float,double,float,double,float,double,float,double,float,double,float,struct):double' (IL size 160)

    File: d:\j\workspace\arm64_cross_c---67295489\src\jit\codegenarm64.cpp Line: 6622
    Image: C:\Users\ddjoin\scratch\CoreRun.exe
```

**Comment by briansull** [comment]
Repro Steps found here:
ARM64 R2R: Assertion failed 'varDsc->lvOnFrame' #5967

**Comment by briansull** [comment]
Fixed with PR  #5993
